kensu/utils/dsl/extractors/external_lineage_dtos.py

```diff
@@ -86,7 +88,11 @@ def report(self, ksu, df_result, operation_type, report_output=False, register_o
         for input_ds in self.inputs:
             extract_ksu_ds_schema(ksu, input_ds, report=True, register_orig_data=True)
         # report output (if needed)
-        result_ds, result_schema = extract_ksu_ds_schema(ksu, df_result, report=report_output, register_orig_data=register_output_orig_data)
```

Comment:
result_ds could be undefined later in code, add **result_ds = None** before

Reply:
ok changed, not used anyway